src/ucs/debug/memtrack.c

```diff
@@ -325,6 +325,19 @@ int ucs_munmap(void *addr, size_t length)
                   length + sizeof(*buffer) + buffer->offset);
 }
 
+char *ucs_strdup(const char *src)
+{
+  char *str;
+  size_t len = strlen(src);
+
+  str = ucs_malloc(len + 1, "ucs_strdup");
```

Comment:
need to add "name" argument to ucs_strdup, and pass it to ucs_malloc, rather than passing "ucs_strdup"